modules/hcp-dc-infraenv.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disconnected/hcp-deploy-dc-bm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-dc-infraenv_{context}"]
+= Creating an InfraEnv resource for the hosted cluster
+
+The `InfraEnv` resource is an Assisted Service object that includes essential details, such as the `pullSecretRef` and the `sshAuthorizedKey`. Those details are used to create the Red Hat Enterprise Linux CoreOS (RHCOS) boot image that is customized for the hosted cluster.
+
+.Procedure
+
+. Create a YAML file with the following information about the `InfraEnv` resource, replacing values as necessary:
```

Comment:
I would remark here:

You could host more than one InfraEnv and each one could adopt certain types of host (more info [here](https://github.com/openshift/assisted-service/blob/master/docs/hive-integration/agent-labels.md)) for example, to divide your server farm between Host with more Ram than X of certain type of disks.

Reply:
I added a couple sentences to stat that you can host multiple InfraEnv resources. Unfortunately, we can't link to the Agent Inventory Labels docs from the official docs.